### DISCUSSION TITLE
CRC Windows | Remote Access

### DISCUSSION THREAD

**Discussion by devopski** [crc-org/crc#3244] [General]
Hello,

I am running CRC on Windows10 with Hyper V. Is there any way to access CRC instance from another PC in the same network? 
I wish I could install CRC on one Windows host and connect to it from another.

**Comment by cfergeau** [comment]
[This thread](https://github.com/code-ready/crc/issues/3245) is about the same question if I'm not mistaken

**Reply by devopski** [reply]
Exactly. Looks like so far we have procedure only for Linux based CRC.

**Reply by gbraad** [reply]
this is based on your own network and forwarding/proxy rules. We have described a semi-supported setup in our documentation based on Linux as that has been the most requested. If you have suggestions on how to offer this on Windows.
